core/model/modx/sources/modfilemediasource.class.php

```diff
@@ -592,6 +592,25 @@ public function checkFiletype($filename) {
         return true;
     }
 
+    /**
+     * Check that an object (directory, file) exists
+     *
+     * @param string $objectPath The object path to check
+     * @param string $objectName The object name displayed in the error message
+     * @return bool
+     */
+    public function checkObjectExist($objectPath, $objectName) {
```

Comment:
If I see a method like this I expect it to return `true` if the object already exists.

Reply:
I agree with Joshua, this should definitely return true if the object exists.

Reply:
But it returns true, if the file or the path exists.

Reply:
That's because @Ruslan-Aleev force-pushed his changes after my comment 😋